mmtk/src/collection.rs

```diff
@@ -39,7 +39,11 @@ impl Collection<V8> for VMCollection {
         }
     }
 
-    fn prepare_mutator<T: MutatorContext<V8>>(_tls: OpaquePointer, _m: &T) {
+    fn prepare_mutator<T: MutatorContext<V8>>(
+        _tls_w: OpaquePointer,
+        _tls_m: OpaquePointer,
```

Comment:
`_tls_m` is the mutator tls and `_tls_w` is the worker tls right?

Reply:
I think it's better to either add a comment or spell out worker/mutator in the variable name. It confused me for a minute...

Reply:
You're right,
Ok, I'll change it to full names

Reply:
@qinsoon @wenyuzhao,

I just thought about similar codes which are copied from`mmtk-core` to bindings.
I thought it may be better if we copy the code comments (e.g. documentation comments) besides the code itself to bindings.
What do you think?

Reply:
What do you mean by copying code to bindings? Did you mean the trait implementations should have comments from the core as well?

Reply:
Yes, that's what I mean